### DISCUSSION TITLE
How to create custom tags in adonis that will work with edge template?

### DISCUSSION THREAD

**Discussion by feniljariwala82** [adonisjs/core#2357] [Help]
I'm implementing RBAC(Role-based authentication system) in adonisjs. And I need to create some basic annotations for my app. Your help will be appreciated.

**Comment by McSneaky** [comment]
Perhaps thease can help little bit
https://preview.adonisjs.com/guides/views/api#registertag
https://github.com/adonisjs/adonisjs.com/blob/develop/docs/docs/guides/views/reference/03-create-custom-tags.md (WIP)
